runtime/gc_base/GCExtensions.cpp

```diff
@@ -295,3 +296,18 @@ MM_GCExtensions::registerScavenger(MM_Scavenger *scavenger)
 	Assert_MM_true(isScavengerEnabled());
 	((MM_StandardAccessBarrier *)accessBarrier)->registerScavenger(scavenger);
 }
+
+void
+MM_GCExtensions::releaseNativesForContinuationObject(MM_EnvironmentBase* env, j9object_t objectPtr)
+{
+#if JAVA_SPEC_VERSION >= 19
+	J9VMThread *vmThread = (J9VMThread *)env->getLanguageVMThread();
+
+	if (verify_continuation_list == getExtensions(env)->continuationListOption) {
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, objectPtr);
+	Assert_MM_true(NULL == continuation);
+	} else {
+		vmThread->javaVM->internalVMFunctions->freeContinuation(vmThread, objectPtr);
```

Comment:
this class should really have javaVM cached
interestingly, it has a javaVM() helper (that fetches it from omrVM each time) and it's used a number of times, but should just be a cached value
but let's be consistent at least and get it using javaVM() helper here, as well
(in future, if there is really a hot path that uses it, we can easily find them all and replace with a chashed value)